pkg/vgmanager/vgmanager_controller.go

```diff
@@ -488,15 +565,50 @@ func (r *VGReconciler) generateVolumeGroupNodeStatus() (*lvmv1alpha1.LVMVolumeGr
 	//lvmvgstatus := vgNodeStatus.Spec.LVMVGStatus
 	var statusarr []lvmv1alpha1.VGStatus
 
+	var isVgExist bool
+
 	for _, vg := range vgs {
+
+		// Add pvs as per volumeGroup CR if path is given add path else add name
+		diskPattern := internal.DiskByNamePrefix
+
```

Comment:
There could be vgs that are not created by LVMO.  Please skip the processing in such a case.

Reply:
I would suggest let's take it up in the next PR. So that QE will have something to test ASAP.

Reply:
It is quite a small change - if !ok { continue }.

Reply:
done